package.json

```diff
@@ -88,6 +89,7 @@
     "karma-coverage-istanbul-reporter": "~3.0.2",
     "karma-jasmine": "~4.0.0",
     "karma-jasmine-html-reporter": "^1.5.0",
+    "madge": "^6.1.0",
```

Comment:
Is this package something we need to include? Considering it was only used to check for circular dependencies and from the sounds of it it wasn't super helpful

Reply:
Nope, meant to remove that. Good catch